website/src/content/concepts/integrate-with-your-own-api.mdx

```diff
@@ -170,6 +201,8 @@ The Exchange token structure consist of **Registered Claim Names** and **Private
 - `iat`: This value represents the time at which the JWT was issued.
 - `type`: This value is a **private claim** and is always set to `exchange`.
 - `<iss>/claims/project_key`: This value represents the Project key that the request is associated to. The claim is a **public claim** prefixed by a collision-resistant namespace, as per the JWT specification.
+- `<iss>/claims/user_permissions` (optional): This value contains a list of the permissions names that are applied to the given logged in user. See [Configuring custom claims](#configuring-custom-claims). The claim is a **public claim** prefixed by a collision-resistant namespace, as per the JWT specification.
```

Comment:
```suggestion
- `<iss>/claims/user_permissions` (optional): Contains a list of permissions names are applied to the logged-in user. See [Configuring custom claims](#configuring-custom-claims). The claim is a **public claim** prefixed by a collision-resistant namespace, as per the JWT specification.
```

Reply:
Here you are suggesting to get rid of the prefix "This value" (as well as for the `project_key` claim) but we use this prefix in other claims:

![image](https://user-images.githubusercontent.com/97907068/187694813-f5aa7630-b76a-4133-bd89-6221e3fc6494.png)

So, if you find that prefix needs to be removed, we would need to remove from all of its 4 usages, right?

Reply:
@CarlosCortizasCT I agree, and that's why added more changes to the PR where I think we can remove `This value`

Reply:
@Anshuman71 Ok. So you provided changes for "iat", "project_key" and "user_permissions" but not for **type** Does it mean you don't think we should change?

Reply:
Yes, because it's a simpler sentence.

Reply:
Updated in this commit: c8053312999955e37e23bc1c73a9fb8896af5314